### PR TITLE
Docs: Add New References

### PR DESCRIPTION
**Pull request by ax3l**
Add missing publications to references in docs and zenodo meta data.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -111,10 +111,16 @@
...
     },
     "references": [
-      "M. Vranic et al. (2015). Classical radiation reaction in particle-in-cell simulations. DOI:10.1016/j.cpc.2016.04.002",
-      "A. Gonoskov et al. (2015). Extended particle-in-cell schemes for physics in ultrastrong laser fields: Review and developments. DOI:10.1103/PhysRevE.92.023305",
       "L. V. Keldysh (1965). Ionization in the field of a strong electromagnetic wave.",
       "D. Bauer and P. Mulser (1999). Exact field ionization rates in the barrier-suppression regime from numerical time-dependent Schr\u00f6dinger-equation calculations. DOI:10.1103/PhysRevA.59.569",
-      "A. Huebl et al. (2015). openPMD 1.0.0 - A meta data standard for particle and mesh based data. DOI:10.5281/zenodo.33624"
+      "R. Pausch et al. (2014). How to test and verify radiation diagnostics simulations within particle-in-cell frameworks. DOI:10.1016/j.nima.2013.10.073",
+      "A. Huebl (2014). Injection Control for Electrons in Laser-Driven Plasma Wakes on the Femtosecond Time Scale. DOI:10.5281/zenodo.15924",
+      "M. Vranic et al. (2015). Classical radiation reaction in particle-in-cell simulations. DOI:10.1016/j.cpc.2016.04.002",
```
Comment (PrometheusPi) [inline .zenodo.json]: This has been published in 2016

Comment (PrometheusPi) [reply .zenodo.json]: Thus if you time ordering the publications, it should go after the openPMD standard.

File: .zenodo.json [hunk 2]
```diff
@@ -111,10 +111,16 @@
...
+      "A. Gonoskov et al. (2015). Extended particle-in-cell schemes for physics in ultrastrong laser fields: Review and developments. DOI:10.1103/PhysRevE.92.023305",
+      "A. Huebl et al. (2015). openPMD 1.0.0 - A meta data standard for particle and mesh based data. DOI:10.5281/zenodo.33624",
```
Comment (PrometheusPi) [inline .zenodo.json]: overpedantic: the official title has a colon no dash. (no need to fix)

File: .zenodo.json [hunk 3]
```diff
@@ -111,10 +111,16 @@
...
+      "A. Matthes et al. (2016). In situ, steerable, hardware-independent and data-structure agnostic visualization with ISAAC. DOI:10.14529/jsfi160403",
+      "A. Huebl et al. (2017). On the Scalability of Data Reduction Techniques in Current and Upcoming HPC Systems from an Application Perspective. DOI:10.1007/978-3-319-67630-2_2",
+      "R. Pausch et al. (2018). Quantitatively consistent computation of coherent and incoherent radiation inparticle-in-cell codes - a general form factor formalism for macro-particles. DOI:10.1016/j.nima.2018.02.020",
```
Comment (PrometheusPi) [inline .zenodo.json]: there is a space missing in `inparticles-...`-> `in particle-...`

File: docs/source/usage/plugins.rst [hunk 4]
```diff
@@ -88,3 +88,42 @@ If you would like to help in developing those classes for a plugin of your choic
...
+        A. Huebl, R. Widera, F. Schmitt, A. Matthes, N. Podhorszki, J.Y. Choi, S. Klasky, and M. Bussmann.
+        *On the Scalability of Data Reduction Techniques in Current and Upcoming HPC Systems from an Application Perspective.*
+        ISC High Performance Workshops 2017, LNCS 10524, pp. 15-29 (2017),
+        `arXiv:1706.00522 <https://arxiv.org/abs/1706.00522>`_, `DOI:10.1007/978-3-319-67630-2_2 <https://doi.org/10.1007/978-3-319-67630-2_2>`_
+
+.. [Pausch2012]
+        R. Pausch.
+        *Electromagnetic Radiation from Relativistic Electrons as Characteristic Signature of their Dynamics*,
+        Diploma Thesis at TU Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Physiker" (2012),
+        https://www.hzdr.de/db/Cms?pOid=38997
```
Comment (PrometheusPi) [inline docs/source/usage/plugins.rst]: Please use the DOI here: `10.5281/zenodo.843510` not the URL

File: docs/source/usage/plugins.rst [hunk 5]
```diff
@@ -88,3 +88,42 @@ If you would like to help in developing those classes for a plugin of your choic
...
+
+.. [Pausch2014]
+        R. Pausch, A. Debus, R. Widera, K. Steiniger, A.Huebl, H. Burau, M. Bussmann, and U. Schramm.
+        *How to test and verify radiation diagnostics simulations within particle-in-cell frameworks*,
+        Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment 740.11, pp. 250-256 (2014)
```
Comment (PrometheusPi) [inline docs/source/usage/plugins.rst]: There is not extra `.11` after `740` this is just the day of publication. (neither the month nor anything relevant)

File: docs/source/usage/plugins.rst [hunk 6]
```diff
@@ -88,3 +88,42 @@ If you would like to help in developing those classes for a plugin of your choic
...
+        `DOI:10.1016/j.nima.2013.10.073 <https://doi.org/10.1016/j.nima.2013.10.073>`_
+
+.. [Pausch2018]
+        R. Pausch, A. Debus, A. Huebl, U. Schramma, K. Steiniger, R. Widera, and M. Bussmann.
+        *Quantitatively consistent computation of coherent and incoherent radiation inparticle-in-cell codes - a general form factor formalism for macro-particles*,
+        Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment 909, 12, pp. 419-422 (2018)
```
Comment (PrometheusPi) [inline docs/source/usage/plugins.rst]: same here , no extra `, 12` (style also differs)

File: docs/source/usage/plugins.rst [hunk 7]
```diff
@@ -88,3 +88,42 @@ If you would like to help in developing those classes for a plugin of your choic
...
+
+.. [Pausch2014]
+        R. Pausch, A. Debus, R. Widera, K. Steiniger, A.Huebl, H. Burau, M. Bussmann, and U. Schramm.
+        *How to test and verify radiation diagnostics simulations within particle-in-cell frameworks*,
+        Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment 740.11, pp. 250-256 (2014)
+        `DOI:10.1016/j.nima.2013.10.073 <https://doi.org/10.1016/j.nima.2013.10.073>`_
+
+.. [Pausch2018]
+        R. Pausch, A. Debus, A. Huebl, U. Schramma, K. Steiniger, R. Widera, and M. Bussmann.
+        *Quantitatively consistent computation of coherent and incoherent radiation inparticle-in-cell codes - a general form factor formalism for macro-particles*,
```
Comment (PrometheusPi) [inline docs/source/usage/plugins.rst]: same typo: 

there is a space missing in inparticles-...-> in particle-...

**Comment by PrometheusPi** [comment]
Since I have currently a update of the radiation plugin documentation open as pull request too, I will add the references there within that pull request #3052.

File: .zenodo.json (hunk 2 above)
Comment (ax3l) [reply .zenodo.json]: fixed with all-version DOI

File: docs/source/usage/plugins.rst (hunk 4 above)
Comment (ax3l) [reply docs/source/usage/plugins.rst]: Cool!

File: docs/source/usage/plugins.rst (hunk 5 above)
Comment (ax3l) [reply docs/source/usage/plugins.rst]: must have mismatched this with the edition when I copied this. will just remove the 11

File: docs/source/usage/plugins.rst (hunk 6 above)
Comment (ax3l) [reply docs/source/usage/plugins.rst]: removed

**Comment by PrometheusPi** [comment]
@ax3l Thanks for the fixes.

**Pull request merged**